layouts/partials/rhds/footer.html

```diff
@@ -46,7 +46,7 @@ <h3 slot="header" data-analytics-text="About Red Hat">About Red Hat</h3>
   </rh-footer-block>
   <rh-footer-block slot="main-secondary">
     <h3 slot="header" data-analytics-text="Subscribe to our newsletter Red Hat Shares">Subscribe to our newsletter, Red Hat Shares</h3>
-    <rh-cta><a href="https://www.redhat.com/en/email-preferences?newsletter=RH-Shares&amp;intcmp=7016000000154xCAAQ" data-analytics-category="Footer|About Red Hat" data-analytics-text="Sign up now">Sign up now</a></rh-cta>
+    <rh-cta><a href="https://www.redhat.com/en/email-preferences?newsletter=RH-Shares&amp;intcmp=7016000000154xCAAQ" data-analytics-linktype="cta" data-analytics-category="Footer|About Red Hat" data-analytics-text="Sign up now">Sign up now</a></rh-cta>
```

Comment:
These where not coming in as CTAs so I added th cta linktype.
Ref: https://github.com/searchdiscovery/redhat-datalayer/blob/main/ClicktoEDDL/clicktoeddl.md#html-code-6